src/main/java/com/answerdigital/answerking/controller/CategoryController.java

```diff
@@ -121,7 +121,9 @@ public ResponseEntity<CategoryResponse> getCategoryById(@PathVariable @NotNull f
             content = { @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class)) })
     })
     @GetMapping("/{categoryId}/products")
-    public ResponseEntity<Collection<ProductResponse>> fetchProductsByCategory(@PathVariable @NotNull final Long categoryId) {
+    public ResponseEntity<Collection<SimpleProductResponse>> fetchProductsByCategory(
+            @PathVariable @NotNull final Long categoryId
```

Comment:
Could we changed this to just 'id' and reflect throughout, just thinking of the specification and consistency! 😄 

```suggestion
            @PathVariable @NotNull final Long id
```

Reply:
I'll make a separate ticket to resolve this issue! I think it'd be better to do it all at once so we don't end up with wildly inconsistent naming.